src/main/scala/com/gu/inspectorlambda/aws/AWSInspector.scala

```diff
@@ -59,6 +59,21 @@ class AWSInspector(val client: AmazonInspector) extends StrictLogging {
     }
   }
 
+  // These implementations can be deleted once the old bad targets and
+  // templates have been removed.
+  def deleteOldAssessmentTarget(name: String): Unit = {
+    val assessmentTargets = getAllAssessmentTargets(None)
+    val matchingAssessmentTargets = assessmentTargets.filter(assessmentTarget => assessmentTarget.getName.equals(name))
+    matchingAssessmentTargets
+      .foreach(assessmentTarget => {
+        logger.info(s"Deleting ${assessmentTarget.getArn}")
+        val deleteAssessmentTargetRequest = new DeleteAssessmentTargetRequest()
+          .withAssessmentTargetArn(assessmentTarget.getArn)
+        client.deleteAssessmentTarget(deleteAssessmentTargetRequest)
+        Thread.sleep(3)
```

Comment:
This is a very long time to wait. I sit really required?

What's the lambda timeout (and will this exceed it)?

Reply:
It is a long time to wait.  I got failures with 1 sec, but not with 3.  So I picked 3.

Total timeout is 5 minutes.  There is also a 10 sec timeout elsewhere.  Altogether, five minutes should be *plenty*.

Reply:
Actually, thinking more about it - I got failures when doing delete/create.  I don't think I will when doing delete only.  I could change to a single sleep after all the deletes are done?

Reply:
Oh wait, Thread.sleep takes millis. Ok, I guess this is fine then, but I'm going to double down on the "is this really required" aspect.